packs/classfeatures/ancestors.json

```diff
@@ -30,6 +30,105 @@
+                            }
+                        ],
+                        "text": "@UUID[Compendium.pf2e.feat-effects.Item.Effect: Curse of Ancestral Meddling]"
+                    },
+                    {
+                        "predicate": [
+                            "oracular-curse:stage:1",
+                            "class:oracle"
+                        ],
+                        "text": "PF2E.SpecificRule.Oracle.CurseSeverity.MinorToModerate"
+                    },
+                    {
+                        "predicate": [
+                            "oracular-curse:stage:2",
+                            "class:oracle",
+                            {
+                                "gte": [
+                                    "self:level",
+                                    11
+                                ]
+                            }
+                        ],
+                        "text": "PF2E.SpecificRule.Oracle.CurseSeverity.ModerateToMajor"
+                    },
+                    {
+                        "predicate": [
+                            "oracular-curse:stage:3",
+                            "class:oracle",
+                            {
+                                "gte": [
+                                    "self:level",
+                                    17
+                                ]
+                            }
+                        ],
+                        "text": "PF2E.SpecificRule.Oracle.CurseSeverity.MajorToExtreme"
+                    },
+                    {
+                        "predicate": [
+                            "class:oracle",
+                            {
+                                "or": [
+                                    {
+                                        "and": [
+                                            "oracular-curse:stage:2",
+                                            {
+                                                "lt": [
+                                                    "self:level",
+                                                    11
+                                                ]
+                                            }
+                                        ]
+                                    },
+                                    {
+                                        "and": [
+                                            "oracular-curse:stage:3",
+                                            {
+                                                "lt": [
+                                                    "self:level",
+                                                    17
+                                                ]
+                                            }
+                                        ]
+                                    },
+                                    "oracular-curse:stage:4"
```

Comment:
We should be able to restructure this along the same lines by checking for not those features.

Reply:
All done